### ISSUE TITLE
Business domain incorrectly blocked

### ISSUE THREAD

**Issue by janjaeschke** [hagezi/dns-blocklists#2084]
### Which AdBlocker/DNS cloud service do you use?

Pi-hole

### NextDNS users only

- [ ] I have disabled the block page in NextDNS and confirm that the problem still occurs!

### With which block list(s) does the problem occur?

Multi PRO

### Which domain(s) should be unblocked?

- netatwork.de
- www.netatwork.de
- *.netatwork.de

### Why should the domain(s) be unblocked?

We are a german business company specialized for Microsoft, E-Mails and IT Security.
As far as I know we do not have any issues publishing unwanted content which should cause a blocking, we are not hosting ads at all. From a technician perspective I see no reason for blocking. 

If you can explain how/why we got added to the list please inform us.
If there is a chance we are added to different lists, please remove us from those too.

**Comment by hagezi** [comment]
Thanks @janjaeschke, via Source 1Hosts, ping @badmojr.
Will be removed from my lists.

**Comment by hagezi** [comment]
@janjaeschke fixed.

**Comment by janjaeschke** [comment]
@hagezi thank you very much for the fast reply.

I will wait a bit and might create another issue for 1Hosts too if badmojr is not reacting here =)
